src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -176,4 +179,16 @@ public void openNotifications() {
     public void toggleLocationServices() {
         CommandExecutionHelper.execute(this, toggleLocationServicesCommand());
     }
+
+    public void toggleWifi() {
```

Comment:
This might be useful to have these methods in the separate `SupportsNetworkStateManagement` interface

Reply:
Okay i will do that :) cheers